test/examples/src/js/UpdateDoc.js

```diff
@@ -36,35 +36,60 @@ const updateDoc = async () => {
       })
     ).result;
 
+    // ==========================================================================
     // Note: for response byte stream use:
-    // const documentAsByteStream = (
-    //     await client.getDocumentAsStream({
-    //       docId: 'example',
-    //       db: exampleDbName,
-    //     })
-    // ).result;
+    /*
+    const documentAsByteStream = (
+      await client.getDocumentAsStream({
+        docId: 'example',
+        db: exampleDbName,
+      })
+    ).result;
+    */
+    // ==========================================================================
 
     // Add Bob Smith's address to the document
     document.address = '19 Front Street, Darlington, DL5 1TY';
 
     // Remove the joined property from document object
     delete document['joined'];
 
-    // Keeping track with the revision number of the document object:
+    // Keeping track with the latest revision number of the document object
+    // is necessary for further UPDATE/DELETE operations:
+    document._rev = (
+      await client.postDocument({
+        db: exampleDbName,
+        document, // _id and _rev MUST be inside the document object
```

Comment:
I think we can remove the comma
```suggestion
        document // _id and _rev MUST be inside the document object
```

Reply:
This is added by prettier on save.